app/src/androidTest/java/com/github/polypoly/app/JoinGameLobbyActivityTest.kt

```diff
@@ -20,24 +20,28 @@ class JoinGameLobbyActivityTest: PolyPolyTest(false, true) {
     fun launchActivity_componentsDisplayed() {
         composeTestRule.onNodeWithTag("gameLobbyCodeField").assertIsDisplayed()
         composeTestRule.onNodeWithTag("JoinGameLobbyButton").assertIsDisplayed()
+
+        Thread.sleep(1000)
         composeTestRule.onNodeWithTag("logo").assertIsDisplayed()
     }
 
     @Test
     fun inputInvalidGameLobbyCode_displayWarningMessage() {
         //TODO: Check for a group code that is not in the DB once we have the queries set
-        composeTestRule.onNodeWithTag("gameLobbyCodeField").performTextInput("polypoly")
+        composeTestRule.onNodeWithTag("gameLobbyCodeField").performTextInput("polpolu")
+        Thread.sleep(1000)
```

Comment:
:face_with_spiral_eyes:

Reply:
This was some hopeless try to make the test pass in cirrus, since I had no clue about Solal’s solution yet :( we all had this problem of tests not running and the common solution was to comment the tests with issues, which I did not like as a solution. I prefere to have this dirty solution for now, while the tests are still getting checked. Also it is marked as an issue to resolve these tests and I believe we talked about turning it into a task later on